src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerLivenessFunctionalTest.groovy

```diff
@@ -11,7 +11,7 @@ class DockerLivenessFunctionalTest extends AbstractGroovyDslFunctionalTest {
             task livenessProbe(type: DockerLivenessContainer) {
                 dependsOn startContainer
                 targetContainerId startContainer.getContainerId()
-                livenessProbe(300000, 30000, 'database system is ready to accept connections')
+                livenessProbe(300000, 1000, 'database system is ready to accept connections')
```

Comment:
There are multiple timeouts in this test class. Let's reduce the number for all of them.

Reply:
On it.